Scripts/Tracking/Follow/ObjectFollow.cs

```diff
@@ -0,0 +1,217 @@
+﻿namespace VRTK.Core.Tracking.Follow
+{
+    using UnityEngine;
+    using UnityEngine.Events;
+    using System;
+    using VRTK.Core.Data.Attribute;
+    using VRTK.Core.Data.Enum;
+    using VRTK.Core.Process;
+    using VRTK.Core.Tracking.Follow.Modifier;
+
+    /// <summary>
+    /// The ObjectFollow mirrors the transform properties of another object based on the given FollowModifier.
+    /// </summary>
+    public class ObjectFollow : SourceTargetProcessor
+    {
+        [Header("Object Follow Settings")]
+
+        [UnityFlag]
+        [Tooltip("The transform properties to follow.")]
+        public TransformProperties follow;
+        [Tooltip("The follow modifier mechanic to apply.")]
+        public FollowModifier followModifier = null;

```

Comment:
I see no need to be explicit about the default value since that's what we'll get without specifying it already.

Reply:
👍